ansible/roles/pgo-operator/templates/pgo.yaml.j2

```diff
@@ -52,12 +52,10 @@ Storage:
 {% endif %}
 {% endfor %}
 DefaultContainerResources: {{ default_container_resources }}
-DefaultLoadResources: {{ default_pgpool_resources }}
```

Comment:
I see this key exists and is blank in the bash installer.

https://github.com/CrunchyData/postgres-operator/blob/ce3dbc158fe78abfd9add5275ac09e813a6b151e/conf/postgres-operator/pgo.yaml#L84

It's still used afaict.

https://github.com/CrunchyData/postgres-operator/blob/ce3dbc158fe78abfd9add5275ac09e813a6b151e/apiserver/loadservice/loadimpl.go#L100-L101